sdk/ai/ai-projects/package.json

```
@@ -0,0 +1,160 @@
+{
+  "name": "@azure/ai-projects",
+  "version": "1.0.0-beta.1",
+  "description": "A generated SDK for ProjectsClient.",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "sideEffects": false,
+  "autoPublish": false,
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
+      "browser",
+      "react-native"
+    ],
+    "selfLink": false
+  },
+  "type": "module",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE"
+  ],
+  "sdk-type": "client",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "src/generated/src/projectsClient.ts",
+        "prefix": "userAgentInfo"
+      },
+      {
+        "path": "src/constants.ts",
+        "prefix": "SDK_VERSION"
+      }
+    ]
+  },
+  "dependencies": {
+    "@azure-rest/core-client": "^2.1.0",
+    "@azure/abort-controller": "^2.1.2",
+    "@azure/core-auth": "^1.6.0",
+    "@azure/core-rest-pipeline": "^1.5.0",
+    "@azure/core-util": "^1.9.0",
+    "@azure/logger": "^1.1.4",
+    "@azure/core-lro": "^2.0.0",
```

this should be using v3 of core-lro, right?
```suggestion
    "@azure/core-lro": "^3.0.0",
```

---

Still using 2.0.0, we would be upgrading to 3.0.0 in next beta 2.